lib/src/cli.rs

```diff
@@ -221,6 +221,22 @@ enum ContainerImageOpts {
         imgref: OstreeImageReference,
     },
 
+    /// Unreference one or more pulled container images and perform a garbage collection.
+    Remove {
```

Comment:
I found it a bit surprising that this does not prune repo objects, or perhaps I'm just coming from a different direction.
I would have expected a `--prune-repo-objects=true` as a default here. Does this no-pruning default follows some similar existing design decision in ostree?

Reply:
So...yes.  It is surprising I agree.  The thing is in ostree there is a separation between a "repo" and a "sysroot".  The repo is a generic thing, whereas sysroot is targeted for booting.

A hack we have today is that it's the *sysroot* code that hardcodes invoking `ostree_repo_prune(REFS_ONLY, 0)` - i.e. delete all commits that are not the tip of a ref.

But...one might want other policies, at least in theory.

And specifically for the container code here, I do envision it potentially being used by e.g. flatpak.  I haven't actually looked recently at flatpak's GC policy...ok there's a rabbit hole there, I am not sure what's going on.